### PR TITLE
Minor Fixes

### PR DESCRIPTION
**Pull request by colinmacdonald**
- Fixes an issue where the image validator cb was being called twice if an image took a long time to load.
- Fixes an issue with checking for `http` existing on a URL in caps

### CODE REVIEW

File: lib/view.js [hunk 1]
```diff
@@ -370,38 +372,42 @@ View.prototype._validateImage = function(imageEl, url, cb) {
   // Fixes an issue in IE9 where onload wouldn't trigger
   imageEl.appendChild(img);
 
-  // Timeout attempt to load image
+  // Timeout after 10 seconds attempting to load image
   var timeoutId = window.setTimeout(function() {
```
Comment (tmyrden) [inline lib/view.js]: Could just make this window.setTimeout(clearImg, 1000)

File: lib/view.js [hunk 2]
```diff
@@ -370,38 +372,42 @@ View.prototype._validateImage = function(imageEl, url, cb) {
...
-      img.parentNode.removeChild(img);
+  function clearImg() {
+    if (validated) {
+      return;
     }
 
+    validated = true;
+
+    window.clearTimeout(timeoutId);
+    img.parentNode.removeChild(img);
```
Comment (tmyrden) [inline lib/view.js:390]: Will this ALWAYS be in the DOM at this point? If not, wrap with an "if (img.parentNode)"

Comment (colinmacdonald) [inline lib/view.js:390]: yes, because clearImg will only get called once due to `validated`

**Comment by tmyrden** [comment]
LGTM pending :white_check_mark:

**Pull request merged**